Task Execution Failure for `:app:mergeDebugJavaResource` with `android-omh-auth`

When using the `android-omh-auth` library, the following error is generated during the build process:

```
org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':app:mergeDebugJavaResource'.
...
Caused by: com.android.builder.merge.DuplicateRelativeFileException: 2 files found with path 'META-INF/DEPENDENCIES'.
Adding a packaging block may help, please refer to
https://developer.android.com/reference/tools/gradle-api/8.5/com/android/build/api/dsl/Packaging
for more information
```

#### Workaround
We have identified a workaround that resolves the issue by modifying the `build.gradle.kts` file at the app level:

```kotlin
packaging {
    resources.excludes.add("META-INF/DEPENDENCIES")
}
```

While this workaround is effective, ideally, users should not have to manually add this configuration to their projects.

#### Request
Please investigate a potential fix for this issue so that users do not need to manually add the packaging block. In the meantime, it would be helpful to document this workaround in the `android-omh-auth` documentation, similar to how it has been documented for `react-native-omh-auth` [here](https://openmobilehub.github.io/react-native-omh-auth/docs/getting-started/#2-files-found-with-path-meta-infdependencies).

--- a/README.md
+++ b/README.md
@@ -231,6 +231,29 @@ the "[creating a custom implementation](https://openmobilehub.github.io/android-
 Please contribute! We will gladly review any pull requests. Make sure to read
 the [CONTRIBUTING](/CONTRIBUTING.md) page first though.
 
+
+## Troubleshooting
+
+### 2 files found with path 'META-INF/*'
+
+> [!CAUTION]
+>2 files found with path 'META-INF/DEPENDENCIES' from inputs:<br/>\- ~/.gradle/caches/transforms-3/974cf06afa1f6a930c75c22e66ec0fcc/transformed/jetified-httpclient-4.5.13.jar<br/>\- ~/.gradle/caches/transforms-3/435f87d6658aae4740f0a56a1a48eac1/transformed/jetified-httpcore-4.4.15.jar
+
+If you see the above error during the android build process, please add the following code snippet to your [**android/app/build.gradle**](https://github.com/openmobilehub/android-omh-auth/blob/main/apps/auth-sample/build.gradle.kts) file:
+
+```gradle title="apps/auth-sample/build.gradle.kts" {4-6}
+android {
+    // ... other build settings
+
+    packagingOptions {
+        resources.excludes.add("META-INF/*")
+    }
+}
+```
+
+Please also check the related [GitHub issue](https://github.com/auth0/Auth0.Android/issues/598) for more context to the problem.
+
+
 ## License
 
 ```

